index.html

```diff
@@ -174,95 +180,122 @@
       </div>
       <section>
         <h2>
-          Syntax
+          Validation
         </h2>
         <p>
-          When URLs are used for payment method identifiers they MUST be
-          absolute URLs including, at most, a <a>scheme</a>, <a>host</a>,
-          <a>port</a> and <a>path</a>. The URL must be a <a>potentially
-          trustworthy URL</a> as defined in the [[SECURE-CONTEXTS]]
-          specification. Identifier URLs MUST have null <a>query</a> and
-          <a>fragment</a> values.
+          The steps to <dfn data-lt="validate|valid">validate a URL-based
+          payment method identifier</dfn> are given by the following algorithm.
+          The algorithm takes a <var>input</var> string as input and returns
+          either true or an error:
         </p>
+        <ol class="algorithm">
+          <li>Let <var>url</var> be the result of running the <a>URL parser</a>
+          on <var>input</var>.
+          </li>
+          <li>If failure, <a>throw</a> a <code>TypeError</code> exception.
+          </li>
+          <li>If running <a>potentially trustworthy URL</a> with <var>url</var>
```

Comment:
How does the developer know where the manifest file is?

This puts the burden on 1 million developers instead of 1 time on the origin. Or am I missing something?

Ian